scripts/msgmerge.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+cd `dirname $0`/..
+for i in modules/mod_ginger*/translations/template/*.pot
+do
+    pushd `dirname $i` > /dev/null
+    if [ -e "../nl.po" ]
+    then
+        echo "Merging into nl.po from '$i'"
+        msgmerge ../nl.po `basename $i` > ../x.po
+        mv ../x.po ../nl.po
```

Comment:
Why not have `../nl.po` as the direct target of `msgmerge` instead of moving?

Reply:
msgmerge doesn't make a new file, so we will be overwriting the input file while we are reading it.

Reply:
You can use `msgmerge -U`.

Reply:
Using `--update` now